pkg/server/image_pull.go

```diff
@@ -455,6 +455,9 @@ const (
 	// targetDigestLabel is a label which contains layer digest and will be passed
 	// to snapshotters.
 	targetDigestLabel = "containerd.io/snapshot/cri.layer-digest"
+	// targetChainLabel is a label which contains layer digests contained in the target
+	// image and will be passed to snapshotters for preparing layers in parallel.
+	targetChainLabel = "containerd.io/snapshot/cri.chain"
```

Comment:
I'd expect this terminology to be reserved for something related to OCI ChainID: https://github.com/opencontainers/image-spec/blob/e562b04403929d582d449ae5386ff79dd7961a11/config.md#layer-chainid

Reply:
Thanks for the feedback. Should we use another word here like `Stack` or `relatedLayers`?

Reply:
Fixed it in the new commit.

Reply:
Curious, can we just make it compatible with OCI ChainID?

Reply:
If not the reason and the differences should be noted as code comment lines.

Reply:
If my understanding is correct...,
Snapshotters requires the list of layer digests in the target image but ChainID is a recursively digested list of layers, with which we cannot restore the original list of layer digest.
So just passing ChainID to snapshotter isn't enough?

Reply:
Or using " "(white space) as a delimiter instead of "," makes the layer digest list compatible to OCI-defined `chain`?